rflx/pyrflx/message.py

```diff
@@ -352,3 +353,66 @@ def __simplified(self, expr: Expr) -> Expr:
         }
 
         return expr.simplified(field_values).simplified(self.__type_literals)
+
+    def parse_from_bytes(self, msg_as_bytes: bytes) -> None:
+
+        msg_as_bitstr = TypeValue.convert_bytes_to_bitstring(msg_as_bytes)
+        current_field_name = self._next_field(model.INITIAL.name)
+        field_first_in_bistr = 0
```

Comment:
Should this have been `field_first_in_bitstr`? I think this would be a more consistent name. While this is only a minor issue I think it can be a source of typo errors.